distributions/Dist_sb6187.py

```diff
@@ -11,7 +11,7 @@ def __init__(self):
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return (math.sin(x)*math.cos(x))+2
+		return (0.0765440563)*((math.sin(x)*math.cos(x))+2)
```

Comment:
you need to use `np.sin` and `np.cos`, the native python math functions won't support broadcasting.